src/include/execution/vm/bytecode_handlers.h

```diff
@@ -136,6 +136,7 @@ INT_TYPES(BITS);
 // ---------------------------------------------------------
 // Memory operations
 // ---------------------------------------------------------
+typedef uint16_t col_id_t;
```

Comment:
This isn't using `STRONG_TYPEDEF`. Please take a look at where `col_id_t` is already defined, and we'll actually need to refactor the signatures of functions that take this as an arg rather than letting it implicitly cast.